scripts/skill_eval/eval.py

```
@@ -0,0 +1,345 @@
+        print(f"  [{sign}] {sc.id:<32}  {sc.title}")
+        if sc.detail:
+            print(f"         {sc.detail}")
+
+
+# ---------- Main ----------
+
+
+def main() -> int:
+    ap = argparse.ArgumentParser(description=__doc__)
+    ap.add_argument("--broker", default="http://localhost:7899")
+    ap.add_argument("--token-file", default="/tmp/wuphf-broker-token-7899")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/wuphf-broker-token-7899"

(S108)